[2024-11-18T12:08:28.753+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T12:08:28.804+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_gbif_data_atropelamento.save_data scheduled__2024-11-18T11:30:00+00:00 [queued]>
[2024-11-18T12:08:28.829+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_gbif_data_atropelamento.save_data scheduled__2024-11-18T11:30:00+00:00 [queued]>
[2024-11-18T12:08:28.830+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 4
[2024-11-18T12:08:28.879+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): save_data> on 2024-11-18 11:30:00+00:00
[2024-11-18T12:08:28.904+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=123) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-18T12:08:28.909+0000] {standard_task_runner.py:72} INFO - Started process 125 to run task
[2024-11-18T12:08:28.909+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'get_gbif_data_atropelamento', 'save_data', 'scheduled__2024-11-18T11:30:00+00:00', '--job-id', '498', '--raw', '--subdir', 'DAGS_FOLDER/gbif_dag.py', '--cfg-path', '/tmp/tmpi7q9oc8i']
[2024-11-18T12:08:28.911+0000] {standard_task_runner.py:105} INFO - Job 498: Subtask save_data
[2024-11-18T12:08:29.063+0000] {task_command.py:467} INFO - Running <TaskInstance: get_gbif_data_atropelamento.save_data scheduled__2024-11-18T11:30:00+00:00 [running]> on host bf677ca5dd6f
[2024-11-18T12:08:29.426+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='get_gbif_data_atropelamento' AIRFLOW_CTX_TASK_ID='save_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T11:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-18T11:30:00+00:00'
[2024-11-18T12:08:29.432+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-18T12:08:29.432+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-18T12:08:29.434+0000] {logging_mixin.py:190} INFO - Current task name:save_data state:running start_date:2024-11-18 12:08:28.805367+00:00
[2024-11-18T12:08:29.434+0000] {logging_mixin.py:190} INFO - Dag name:get_gbif_data_atropelamento and current dag run status:running
[2024-11-18T12:08:29.435+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T12:08:29.583+0000] {minio.py:40} INFO - created gbif_data/atropelamento_data/2024/11/18/atropelamento_occurrences_120820.json object; etag: 60912e889a13d75fc31f04555f4db86e, version-id: None
[2024-11-18T12:08:29.584+0000] {minio.py:42} INFO - Data successfully uploaded to MinIO in bucket environbit under gbif_data/atropelamento_data/2024/11/18/atropelamento_occurrences_120820.json
[2024-11-18T12:08:29.585+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-18T12:08:29.615+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-18T12:08:29.616+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=get_gbif_data_atropelamento, task_id=save_data, run_id=scheduled__2024-11-18T11:30:00+00:00, execution_date=20241118T113000, start_date=20241118T120828, end_date=20241118T120829
[2024-11-18T12:08:29.659+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-18T12:08:29.659+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-18T12:08:29.660+0000] {logging_mixin.py:190} INFO - Dag name:get_gbif_data_atropelamento queued_at:2024-11-18 12:08:17.123957+00:00
[2024-11-18T12:08:29.661+0000] {logging_mixin.py:190} INFO - Task hostname:bf677ca5dd6f operator:_PythonDecoratedOperator
[2024-11-18T12:08:29.736+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-18T12:08:29.787+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-18T12:08:29.801+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
